### PR TITLE
Event tab bar translate value and constant

### PR DESCRIPTION
**Pull request by junzai97**
One can import the **eventTabList** and applied as the **itemList** value in CustomTab.

```
import { eventTabList } from 'app/shared/util/tab.constants.ts';
import { CustomTab } from 'app/shared/components/customTab/custom-tab';

//Rendering customTab
<CustomTab currentTab="" tabList={ eventTabList } />
```

### CODE REVIEW

**Comment by kachun333** [comment]
please only merge this after #26 is merged.

File: src/main/webapp/app/shared/util/tab.constants.ts
Comment (HowardYaw) [inline src/main/webapp/app/shared/util/tab.constants.ts]: why the tabRoute not with prefix "/event/"  ❓ 

and this tabRoute should synchronize with the actual route of the page
there are existing pages either follow the one 
or we need to change that Route path also

Comment (junzai97) [reply src/main/webapp/app/shared/util/tab.constants.ts]: The tabRoute prefix is resolved in 3348d0f8db46f3f45240588c3776cdd7a27b074e

For the existing pages, will like to discuss tonight.

**Review by HowardYaw — APPROVED** [review]

**Review by kachun333 — APPROVED** [review]

**Pull request merged**
